doc/examples/fiber_to_bundle_coherence.py

```diff
@@ -0,0 +1,280 @@
+# -*- coding: utf-8 -*-
```

Comment:
Woops, I uploaded the wrong figure actually. I've now added ```fbc_illustration.png``` to doc/examples_built/_static, which is the same folder where ```stochastic_process.jpg``` was also placed. Let me know if this is OK.